tests/Doctrine/Tests/DBAL/Functional/Schema/ComparatorTest.php

```diff
@@ -25,15 +25,31 @@ protected function setUp()
         $this->comparator    = new Comparator();
     }
 
-    public function testDefaultValueComparison()
+    /**
+     * @param mixed $value
```

Comment:
Does it make sense to be a bit more explicit here (as in `int|bool`)?

Reply:
It could be used with other types later, so I'd leave it as is.